lib/custom_widgets/title_widget.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:movie_db/text_styles/custom_text_style.dart';
+
+import '../entities/Movie.dart';
+
+class TitleWidget extends StatelessWidget {
+  const TitleWidget({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
```

Comment:
This widget doesn't require the whole movie eentity, but only the title. Please change this! 
Always try to leave the widgets as more generic as possible. In this case, that is only useful for a Movie entity.

Reply:
done